comment_parser/parsers/python_parser.py

```diff
@@ -19,11 +36,24 @@ def extract_comments(code):
   Raises:
     tokenize.TokenError
   """
+  triplequotes = ['"""', "'''"]
+  multicommprevnums = [tokenize.NEWLINE, tokenize.NL, tokenize.INDENT, tokenize.DEDENT]
```

Comment:
Could we call this something like "breaks" instead? multicommprevnums is quite a long variable name.